src/SOS/SOS.Package/SOS.Package.csproj

```diff
@@ -0,0 +1,120 @@
+﻿<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Project>
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <AssemblyName>Microsoft.NETCore.SOS</AssemblyName>
+    <Description>.NET Core SOS binaries</Description>
+    <PackageReleaseNotes>$(Description)</PackageReleaseNotes>
+    <IsPackable>true</IsPackable>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <SOSPackagePathPrefix>tools</SOSPackagePathPrefix>
+    <GalleryManifestName>$(ArtifactsPackagesDir)\GalleryManifest.xml</GalleryManifestName>
+    <BeforePack>GenerateGalleryZip</BeforePack>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <None Include="$(GalleryManifestName)" Pack="true" Visible="false">
+      <PackagePath>$(SOSPackagePathPrefix)</PackagePath>
+    </None>
+  </ItemGroup>
+
+  <Import Project="$(MSBuildThisFileDirectory)..\..\sos-packaging.props" />
+
+  <Import Project="Sdk.targets" Sdk="Microsoft.NET.Sdk" />
+
+  <Target Name="Build" />
+
+  <Target Name="GenerateGalleryManifest" DependsOnTargets="GetAssemblyVersion;AddSourceRevisionToInformationalVersion">
+    <PropertyGroup>
+      <GalleryManifestLines>
+<![CDATA[
+<?xml version="1.0" encoding="utf-8"?>
+<ExtensionPackage>
+    <Name>SOS</Name>
+    <Version>$(FileVersion)</Version> 
+    <Description>Debugging aid for .NET Core programs and runtimes</Description>
+    <Components>
+        <BinaryComponent Name="sos" Type="Engine">
+            <Files>
+                <File Architecture="amd64" Module="win-x64\sos.dll" />
```

Comment:
Are these paths assumed to be from the root, or from the tools folder? Before these folders we inside the version folder and the manifest was at the same depth as the version folder. Now there is no version folder, but a tools folder, and the manifest is inside the tools folder.

Reply:
They are relative to manifest file itself.  So the gallery zip file, the manifest is in the version folder (with win-x64, etc. under that). For the package it is in the tools directory (with the win-x64, etc. under that). The debugger team seems to be fine with the layout of the nupkg.

Reply:
These are old and new zip files generated. The rid folders are not along the manifest so that's why I asked. Also, in one case we packed with the rid, in the other with the architecture. If this doesn't matter, just disregard this.
![image](https://user-images.githubusercontent.com/19413848/61235437-647d4480-a6ea-11e9-9eb0-ae1a71af947f.png)
![image](https://user-images.githubusercontent.com/19413848/61235448-6ba45280-a6ea-11e9-8721-541231f2c839.png)

Reply:
The sub-directory doesn't matter as long as the path in the manifest matches (and it does).

Reply:
Thanks for lcarifying this